gnomad_qc/v3/create_release/get_het_nonref_variants_to_fix.py

```diff
@@ -0,0 +1,183 @@
+import argparse
+import logging
+
+import hail as hl
+
+from gnomad.utils.slack import slack_notifications
+from gnomad.utils.sparse_mt import densify_sites
+from gnomad.utils.vcf import SPARSE_ENTRIES
+
+
+from gnomad_qc.v3.resources.basics import get_gnomad_v3_mt
+from gnomad_qc.v3.resources.release import release_sites
+from gnomad_qc.v3.resources.annotations import get_info, last_END_position
+from gnomad_qc.slack_creds import slack_token
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("get_impacted_variants")
+logger.setLevel(logging.INFO)
+
+
+def get_het_non_ref_impacted_var(
+    mt: hl.MatrixTable, info_ht: hl.Table, freq_ht: hl.Table
+) -> None:
+    """
+    Filter to variants where homalt hotfix incorrectly adjusts het nonref genotype calls.
+
+    :param hl.MatrixTable mt: Raw, split MatrixTable annotated with original genotype het nonref status.
+    :param hl.Table info_ht: Info Hail Table containing AS_lowqual information.
+    :param hl.Table freq_ht: Hail Table containing 455K frequency information.
+    :return: None
+    """
+
+    logger.info("Removing low qual variants and star alleles...")
```

Comment:
I think you mentioned that the v3 MT doesn't have star alleles?

Reply:
Ah yes, I don't need this, I think just a copy over from your script